### PR TITLE
Add extra data to LinkedinUser

### PR DESCRIPTION
**Pull request by eminetto**

### CODE REVIEW

**Comment by eracle** [comment]
I will merge it soon. Thanks

File: linkedin/spiders/selenium.py [hunk 1]
```diff
@@ -119,7 +119,9 @@ def extracts_see_all_url(driver):
 
     a_elem = driver.find_element_by_link_text(see_all_ex_text)
     see_all_url = a_elem.get_attribute('href')
-
+    # see_all_url = 'https://www.linkedin.com/search/results/people/?company=&facetPastCompany=%5B%221235825%22%5D&firstName=&lastName=&origin=FACETED_SEARCH&school=&title='
```
Comment (eracle) [inline linkedin/spiders/selenium.py]: Shall we clean unused comments?

File: Makefile [hunk 2]
```diff
@@ -5,7 +5,7 @@ view:
 	vinagre localhost:5900
 
 companies:
-	scrapy crawl companies -a selenium_hostname=localhost -o users.csv
+	scrapy crawl companies -a selenium_hostname=localhost -o users.jsonlines
```
Comment (eracle) [inline Makefile]: is this change needed?

Comment (eminetto) [inline Makefile]:
```suggestion
	scrapy crawl companies -a selenium_hostname=localhost -o users.csv
```
